lib/bundler/resolver.rb

```diff
@@ -342,15 +342,17 @@ def verify_gemfile_dependencies_are_found!(requirements)
         elsif requirement.source
           name = requirement.name
           versions = @source_requirements[name][name].map(&:version)
+          platform = requirement.__platform
           message  = String.new("Could not find gem '#{requirement}' in #{requirement.source}.\n")
-          message << if versions.any?
-                       "Source contains '#{name}' at: #{versions.join(", ")}"
-                     else
-                       "Source does not contain any versions of '#{requirement}'"
-                     end
+          if versions.any?
+            message << "Source contains '#{name}' at: #{versions.join(", ")}"
+            message << " #{platform}" unless platform.nil? || platform == "ruby"
```

Comment:
By `spec`, do you mean `requirement` in the context of the `verify_gemfile_dependencies_are_found!` method? Looking at this method, it seems to compose an error message and then raise a `GemNotFound` error with that error message when a requirement is not fulfilled. 

So, if you mean print the platforms for each `requirement`, I don't think that'd be necessary since only the first non-fulfilled `requirement` would be displayed/cause an error. Please let me know if I'm mixing up terminology/concepts here.